ldj-readme-renderer/README_template.md

```diff
@@ -0,0 +1,66 @@
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=coverage)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
+[![Technical Debt](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=sqale_index)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
+[![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=reliability_rating)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
+[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=Ali-chakaroun_ISEP-LivingDocumentation&metric=sqale_rating)](https://sonarcloud.io/summary/new_code?id=Ali-chakaroun_ISEP-LivingDocumentation)
+# Living Documentation Java
+
+This project is an extension of the [Living Documentation][ldoc] set of tools to
+Java. As part of the Living Documentation toolchain, it can be used to generate
+documentation and diagrams from source code so that documentation is always
+up-to-date.
+
+## State of the project
+
+As of right now, this project is linked to a student assignment originating from the University of
+Twente and InfoSupport. The aim of the project is to make the [LivingDocumentation][ldoc] project
+capable to handle multilingual projects. One aspect of this is to make LivingDocumentation
+available for the Java environment; which is the purpose of this repository.
+
+Within the aspect of making LivingDocumentation available for the Java environment, two features
+can be identified (which are copied from [LivingDocumentation][ldoc] description):
+
+* [**Analyzer**](#analyzer): A tool to analyze Java projects.
+* [**Libraries**](#support-libraries): Assists in generating applications that can create plain text files such as MarkDown, AsciiDoc, PlantUML, Mermaid, and more.
+
+> **Note**: As this repository is currently association to a University assignment, it is unfortunately not possible to directly contribute to this project. 
```

Comment:
Aww, are you sure about this? I would think we'd gladly take patches, if people have them.

Reply:
I am not sure about this as we are graded on how we work as a project group and the work we deliver should be our own. 
On one hand, it could be considered positive if we manage an open-source project correctly resulting in a better product for Info Support. On the other hand, other people should probably not take over our work.
But again, this is a guess from my side on how Petra would view this (maybe discuss this sometime with Petra and others?)

Reply:
I created a general section about this as discussed with Petra